### ISSUE TITLE
Resampling strategy used when predicting with the test set ?

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#278]
https://github.com/automl/Auto-PyTorch/blob/3413bc32758b4be47b4511a09b64aff4f3a8e005/autoPyTorch/pipeline/base_pipeline.py#L167-L185

https://github.com/automl/Auto-PyTorch/blob/3413bc32758b4be47b4511a09b64aff4f3a8e005/autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py#L148-L153

https://github.com/automl/Auto-PyTorch/blob/3413bc32758b4be47b4511a09b64aff4f3a8e005/autoPyTorch/datasets/base_dataset.py#L83-L87

I am looking at the code and I am getting the impression that we are actually subsampling whatever we give to the predict method, since the `holdout_strategy` is not None by default in `BaseDataset`

**Comment by ravinkohli** [comment]
actually no, we are using the train tensors to input data to the data loader [here](https://github.com/automl/Auto-PyTorch/blob/3413bc32758b4be47b4511a09b64aff4f3a8e005/autoPyTorch/datasets/base_dataset.py#L201). The Holdout splits are stored in self.splits and we are not using them here.
